src/nft/components/profile/list/NFTListingsGrid.tsx

```diff
@@ -86,32 +127,74 @@ const RowDivider = styled.hr`
 export enum SetPriceMethod {
   SAME_PRICE,
   FLOOR_PRICE,
-  PREV_LISTING,
+  LAST_PRICE,
+  CUSTOM,
 }
 
 export const NFTListingsGrid = ({ selectedMarkets }: { selectedMarkets: ListingMarket[] }) => {
   const sellAssets = useSellAsset((state) => state.sellAssets)
-  const [globalPriceMethod, setGlobalPriceMethod] = useState<SetPriceMethod>()
+  const [globalPriceMethod, setGlobalPriceMethod] = useState(SetPriceMethod.CUSTOM)
   const [globalPrice, setGlobalPrice] = useState<number>()
+  const [showDropdown, toggleShowDropdown] = useReducer((s) => !s, false)
```

Comment:
tech debt: a lot of this dropdown boilerplate is redundant across both components. I feel like this could be added to the dropdown component instead

Reply:
I specifically didn't include some logic such as its position and orientation in the dropdown to keep it as light weight and as flexible as possible. Once issue I had with the old dropdown is that it was very rigid and trying to add any level of customization required a lot of overwriting. The trade-off is a little more boilerplate when implementing which I personally prefer.